openelevationservice/server/api/querybuilder.py

```diff
@@ -31,6 +33,91 @@ def _getModel(dataset):
     return model
 
 
+def format_PixelAsPoints(result_pixels):
+    # format: [ ('(0101000020E61000000000000000C05240D169039D36003D40,202,1,2)',) , ...
+    points = []
+    heights = []
+    for pixel in result_pixels:
+        subcolumns = pixel[0].split(",")
+        point = subcolumns[0][1: ]
+        if point not in points:
```

Comment:
Can be points a set? 'in' with list is O(n) and with set is O(1) in average case.
https://wiki.python.org/moin/TimeComplexity

Reply:
They are not needed at the end, adding DISTINCT statement to the SQL query is better.